app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -342,6 +361,22 @@ class BrowserFragment : BaseBrowserFragment(), BackHandler {
         }
     }
 
+    private fun updateTrackingProtectionUIState(session: Session) {
```

Comment:
@Amejia481 this is what I was trying, but was seeing that overlap issue I sent you. This also only works when the toggle is clicked, not when a session is resumed that is an exception :/

Reply:
Cool, I'm on it!